water_cluster_record.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[2]:
+
+
+from wrap_pyscf import *
```

Comment:
Even though this is right in the virtual environment where it should be, I get an error in the notebook from this line about the module not existing.

Reply:
Your PR more-or-less works (I get some errors from further down in the script) for me so it's got to be something about your setup. My guess is this is still a filesystem issue, which is something I can't really help you with since I have no experience running Linux in Windows. One thing you may want to consider is dual booting. Basically you can install Ubuntu next to Windows on your computer. We could then use the Ubuntu partition for research.